test/fluree/db/json_ld/api_test.cljc

```diff
@@ -678,4 +678,53 @@
                       "rdf:type" ["sh:NodeShape"],
                       "sh:targetClass" {"id" "schema:Person"},
                       "sh:property" {"id" "_:f211106232532994"}}]
-                    @(fluree/query (fluree/db loaded2) property-query)))))))))
+                    @(fluree/query (fluree/db loaded2) property-query)))))))
+    (testing "can load after deletion of entire subjects"
+       (let [conn              @(fluree/connect
+                                  {:method :memory
+                                   :defaults
+                                   {:context      test-utils/default-context
+                                    :context-type :keyword}})
+             ledger-alias      "tx/delete"
+             ledger            @(fluree/create conn ledger-alias {:defaultContext ["" {:ex "http://example.org/ns/"}]})
+             db1               @(fluree/stage
+                                  (fluree/db ledger)
+                                  {:graph
+                                   [{:id                 :ex/fluree
+                                     :type               :schema/Organization
+                                     :schema/description "We ❤️ Data"}
+                                    {:id                 :ex/w3c
+                                     :type               :schema/Organization
+                                     :schema/description "We ❤️ Internet"}
+                                    {:id                 :ex/mosquitos
+                                     :type               :ex/Monster
+                                     :schema/description "We ❤️ human blood"}
+                                    {:id                 :ex/kittens
+                                     :type               :ex/Animal
+                                     :schema/description "We ❤️ catnip"}]})
+             description-query '{:select {?s [:id]}
+                                 :where  [[?s :schema/description ?description]]}
+             _                 @(fluree/commit! ledger db1)
+             loaded1           @(fluree/load conn ledger-alias)
```

Comment:
You may want to use `test-utils/retry-load` here and for the other `load` calls or this can get into a race condition. Sometimes it still happens anyway for reasons I haven't fully figured out yet, but this at least cuts down on it a good deal.

Reply:
Ah yes, good call. There are actually some other, unrelated tests above this that also use regular `load` (which I co-wrote 😅), I've just pushed a change to fix all of them.